package/graph.py

```diff
@@ -1,38 +1,69 @@
 import os
 import subprocess
 import sys
+from datetime import datetime
 
 
-def display(snippet_name):
+def copy_to_clipboard(text):
+    """
+    Copies the provided text to the clipboard.
+    
+    Args:
+        text (str): The text to copy to the clipboard.
+
+    Raises:
+        OSError: If the operating system is unsupported.
+    """
+    # Linux
+    if "linux" in sys.platform:
+        subprocess.run(
+            ["/usr/bin/xclip", "-selection", "clipboard"],
+            input=text.strip().encode(),
+            check=True,
+        )
+    # Windows
+    elif "win32" in sys.platform:
+        subprocess.run(
+            ["C:\\Windows\\System32\\clip.exe"],
+            input=text.strip().encode(),
+            check=True,
+        )
+    else:
+        raise OSError("Unsupported operating system")
```

Comment:
Add these following code snippet as well.
```suggestion
        # macOS
        elif "darwin" in sys.platform:
            subprocess.run(
                ["/usr/bin/pbcopy"],  # Full path to pbcopy for macOS
                input=text.strip().encode(),
                check=True,
            )
        else:
            raise OSError("Unsupported operating system")
```

Reply:
Done